### ISSUE TITLE
[E2E] [KYC] Individual verification with IDnow 

### ISSUE THREAD

**Issue by krzkaczor** [Neufund/platform-frontend#753]
Use special api provided by backend team to control verification status. Test things like failed verification as well.

**Comment by Moejoe90** [comment]
Blocked by https://github.com/Neufund/platform-backend/issues/1320

**Comment by desfero** [comment]
Duplicate of #3818
